spec/spec_helper.rb

```diff
@@ -1,6 +1,5 @@
 # encoding: utf-8
 require 'simplecov'
-SimpleCov.minimum_coverage 96.45
```

Comment:
This was too abstract a number based on current coverage. Also, current test unit tests are not really helping the protect SAML 2.0 workflow. Let’s postpone it until we convert the gem more focused to the SAML 2.0 protocol.

Reply:
maybe it was just too precise.. you could have dropped the number to the new coverage instead of removing the line

Reply:
<img width="885" alt="image" src="https://github.com/user-attachments/assets/a15fdbfb-c065-42c5-9d37-0d8da8511ab2">

Reply:
I thought that. But most of the unit tests were meaningless not checking the logic regarding SAML 2.0.
I think once we have proper SAML 2.0 workflow tests and we can re-introduce it.